wordpress-mobile/WordPress-Android
# Fix merge conflicts related to compose buttons

This PR manually resolves the merge conflicts related to `PrimaryButton` and `SecondaryButton`.

To test:
- **Verify** CI is green
- **Verify** manually there's  no visual regression on the unintended areas of impact

## Regression Notes
1. Potential unintended areas of impact
   Screens where the buttons are used: `Jetpack Migration Welcome Screen`, `Jetpack Static Poster`, etc.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
   Manual testing.

3. What automated tests I added (or what prevented me from doing so)
   N/a

PR submission checklist:
